src/redux/news/news.sagas.js

```diff
@@ -0,0 +1,40 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { setNews } from './news.actions';
+import getItems from '../../services/getItems';
+import { NEWS_LOADED } from './news.types';
+
+function* handleNewsLoad() {
+  const news = yield call(
```

Comment:
Wrap it in `try { ... } catch` block to handle errors from server

Reply:
done